packages/components/autocomplete/src/multi-autocomplete.tsx

```diff
@@ -150,6 +158,7 @@ export const MultiAutocomplete = forwardRef<MultiAutocompleteProps, "input">(
     )
     let {
       className,
+      clearable = true,
```

Comment:
Change `clearable = true` to `clearable`.

Reply:
fixed.